src/profile/helpers/__tests__/updateMutationVariables.test.ts

```diff
@@ -97,33 +112,12 @@ const formValues: FormValues = {
   firstName: 'Teemu',
   lastName: 'Testaaja',
   profileLanguage: Language.FINNISH,
-  primaryEmail: {
-    email: 'ensimmainen@testi.fi',
-    primary: true,
-    id: '123',
-    emailType: EmailType.OTHER,
-    __typename: 'EmailNode',
-  },
-  emails: emails,
-  phones: phones,
-  primaryPhone: {
-    id: '123',
-    phone: '0501234567',
-    phoneType: PhoneType.OTHER,
-    primary: true,
-    __typename: 'PhoneNode',
-  },
-  addresses: addresses,
-  primaryAddress: {
-    id: '123',
-    primary: true,
-    address: 'Testikatu 55',
-    city: 'Helsinki',
-    countryCode: 'FI',
-    postalCode: '00100',
-    addressType: AddressType.OTHER,
-    __typename: 'AddressNode',
-  },
+  primaryEmail: cloneObject(emails[0]),
```

Comment:
Could this clone the `primaryEmail`?

Reply:
Tests might break, if data not in the arrays (emails, addresses, phones) is used here where test data is created.  "primaryEmail/Phone/Address" are currently used in the array, but that could change or their data could be used with modifications. 

Bottom line: If arrays change, the changes should not break the tests.